README.md

```diff
@@ -286,6 +243,4 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
 
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ ] I would kindly suggest you work on the MIT license aspect of your project as the link is currently not leading to your license file and it is throwing error 404- Page not Found. Having a working link will make your README file more descriptive and professional. kindly work on it.

MIT link 👇 

![lic](https://user-images.githubusercontent.com/78634841/236939095-fa7aa0f1-d41c-4193-a8d5-357c9fa91361.jpg)